src/app/components/shell/shell.js

```diff
@@ -9,22 +10,34 @@ import zendesk from '~/helpers/zendesk';
     el: 'body',
     template: template,
     regions: {
+        loader: '#loader',
         main: '#main',
         header: '#header',
         footer: '#footer'
     }
 })
+
 class AppView extends BaseView {
 
     constructor() {
         super();
 
+        this.loader = loader;
         this.header = header;
         this.footer = footer;
 
         this.render();
     }
 
+    preLoader() {
+        setTimeout(() => {
+            document.body.classList.remove('no-scroll');
+        }, 1000);
+        setTimeout(() => {
+            document.body.classList.add('loaded');
+        }, 2000);
```

Comment:
Is this just a temporary hack?  A 2 second delay on loading is probably enough to make it look like the site has loaded on machines near the server and with good internet connections, but it's not an accurate representation of the loading state.